src/lib/optimizer/strategy/predicate_placement_rule.cpp

```diff
@@ -16,6 +16,22 @@
 #include "operators/operator_scan_predicate.hpp"
 #include "statistics/cardinality_estimator.hpp"
 
+using namespace opossum;  // NOLINT
+
+namespace {
+
+// TODO
+class TemporaryRootNode : public LogicalPlanRootNode {
+ public:
+  explicit TemporaryRootNode() : LogicalPlanRootNode() {}
+
+  std::string description(const DescriptionMode mode = DescriptionMode::Short) const override {
+    return "[TemporaryRootNode]";
+  }
+};
+
+}
+
```

Comment:
Do you think this is appropriate, or shall we add a regular LQP class (which requires adding case-stmts in our switch-case blocks)?